### ISSUE TITLE
[Feature] add all (not just value) podcasts from Podcast Index

### ISSUE THREAD

**Issue by woiwoiwoiwoi** [breez/breezmobile#1251]
I read in the [docs ](https://doc.breez.technology/How-do-I-add-my-podcast-to-Breez.html) that it was by design to only add those podcasts from the Podcast Index to Breez that "added a Lightning value block to their feeds." Unfortunately, I don't think this is a good solution.

Taking me as an example: I am an avid podcast fan and listener and I'm subscribed to dozens of them. I decided to switch to a Podcasting 2.0 app and chose Breez (open source, non-custodial), but realised I cannot. I am required to search for every podcast on Breez and in the unlikely (most podcasts are not 2.0) event that it's found, great. But if it's not found (most of the time), I need to keep listening to that one on my old podcasting app. This is horrible UX and a reason to not use Breez for podcasts.

Is there a chance this will be changed?

**Comment by kingonly** [comment]
Nope. We're not going to change it.
Breez is primarily about promoting Lightning, and our main goal is to promote Lightning.
We understand that this the optimal UX for some users, on the other hand we've managed to get some podcast on the platform because of that. Apologies.

**Comment by woiwoiwoiwoi** [comment]
Thanks for responding and the explanation. I think I understand where you're coming from, but must say I disagree. I'm a Bitcoiner (for many years now), I try to promote Lightning as much as I can (I started with a selfhosted Lightning node maybe two years ago), and I listen to a lot of podcasts. I've tried out the value 4 value functionality of Breez over the past few days and I love it.

But I will still probably stop using it and use something else instead (checking out Podverse and Fountain right now, both unfortunately custodial afaik).

I believe you don't win new users, and you won't promote Lightning successfully, with an inferior product (from the user perspective). I really don't see how potential Lightning users would be turned off by finding all their podcasts on Breez instead of having to use two apps.

Regarding those podcasts you mentioned: are they really not available anywhere else (via the Podcast Index or else)?
